### PR TITLE
Update Polish translations

### PR DESCRIPTION
**Pull request by carno**

### CODE REVIEW

File: po/pl.po [hunk 1]
```diff
@@ -1667,6 +1661,9 @@ msgid ""
 "exits and comments will be deleted. Press Enter to continue or Ctrl+C to "
 "abort"
 msgstr ""
+"OSTRZEŻENIE: Znaleziono kometarz w `%s'/ Plik kolejki jest nadpisywany kiedy "
```
Comment (Minoru) [inline po/pl.po:1664]: Is the slash a typo?

```suggestion
"OSTRZEŻENIE: Znaleziono kometarz w `%s'. Plik kolejki jest nadpisywany kiedy "
```

File: po/pl.po [hunk 2]
```diff
@@ -360,26 +353,30 @@ msgid "%s is inaccessible and can't be created\n"
...
-msgstr "BŁĄD: Musisz ustawić `cookie-cache` aby korzystać z NewsBlur.\n"
+msgstr "BŁĄD: Musisz ustawić `miniflux-url` aby korzystać z Miniflux.\n"
 
 #: src/controller.cpp:307
 msgid ""
 "ERROR: You must set `miniflux-login` and one of `miniflux-password`, "
 "`miniflux-passwordfile` or `miniflux-passwordeval` to use Miniflux\n"
 msgstr ""
+"BŁĄD: Musisz ustawić `miniflux-login` oraz jeden z `miniflux-password`, "
+"`miniflux-passwordfile` olub`miniflux-passwordeval` aby korzystać z "
```
Comment (Minoru) [inline po/pl.po:365]: I think a space is missing here:

```suggestion
"`miniflux-passwordfile` olub `miniflux-passwordeval` aby korzystać z "
```

**Review by Minoru — CHANGES_REQUESTED** [review]
Thanks! I think I spotted a few typos, please take a look.

File: po/pl.po (hunk 1 above)
Comment (carno) [reply po/pl.po:1664]: Indeed, what is more it should be kome**n**tarz :facepalm:

File: po/pl.po (hunk 2 above)
Comment (carno) [reply po/pl.po]: And a needles `o` to start with :facepalm:

**Review by Minoru — APPROVED** [review]
Thanks!

**Pull request merged**